msteams-platform/m365-apps/overview-combining-add-in-and-teams-app.md

```diff
@@ -0,0 +1,85 @@
+
+However, we can make some general recommendations about combining an add-in with a Teams app.
+
+## Learn about the manifest and structure of a combined Teams app and Office add-in
+
+Before you start work merging your two existing extensions, get a sense of how to merge the two types of extensions by creating a new Teams app and a new add-in in Teams toolkit and then following the instructions in [Add an Outlook Add-in to a Teams app](combine-office-add-in-and-teams-app.md) to combine them. Pay close attention to the following:
+
+- How the single unified manifest is created.
+- The differences in how the add-in and the Teams app are sideloaded and debugged.
+
+## Follow proper principles for combining Teams apps and add-ins
+
+When you are ready to merge your existing add-in and Teams app, follow these principles:
+
+- If the add-in uses an XML manifest, convert it to use the unified manifest for Microsoft 365 as described in [Convert an add-in to use the unified manifest for Microsoft 365](/office/dev/add-ins/develop/convert-xml-to-json-manifest) before you combine it with a Teams app.
+- Use the Teams app project as the base project for the merged application.
+- Keep configuration files that are applicable to both the add-in and Teams app in the root of the project.
+- Move source files and configuration files that are only applicable to the Teams app into a subfolder off the root of the project. Similarly, copy source and configuration files that are only used by the add-in from the existing add-in to a different subfolder off the root of the combined project.
+- Use the same base domain for the built files of both the add-in and Teams app, for example contoso.com.
+- Merge the two manifests into a single manifest. Use the process you went through in step 1 as a guide.
+- The [id](../resources/schema/manifest-schema.md#id) property in the new manifest should be the same value as the "id" property in the original Teams app manifest.
+- Raise the value of the [version](../resources/schema/manifest-schema.md#version) property; for example, raise "1.0.0" to "1.1.0".
+- Add an [extensions.alternates](../resources/schema/manifest-schema.md#extensionsalternates) property to the manifest and configure it to hide the original Office add-in in versions of Office that support combining an add-in and a Teams app. For more information, see [Manage new and old versions of an add-in](/office/dev/add-ins/concepts/duplicate-legacy-metaos-add-ins).
+
+## Publish the combined Teams app and Office add-in
+
+To publish the combined app, treat it like an update to the Teams app. For more information, see [Publish updates to your app](../concepts/deploy-and-publish/appsource/post-publish/overview.md#publish-updates-to-your-app).
+
+When the update is accepted, how it becomes available to end users varies. The following are the general principles.
+
```

Comment:
I would put it like this - 

1. User acquires the new app from Outlook or Teams
   * New app becomes available for use (is deployed) across hubs. In case this does not happen please see the admin policies for that hub
    * If the hide property has been populated in the manifest of the new app, the old XML add-in will get hidden in Outlook and end user will see only one app at a given point in time. 
    
2. Admin deploys the new app on Outlook 
   * New app will be available for use to the users on Outlook. User may have to acquire the app / app update separately on Teams (this behavior will change soon and deployment on one hub will be reflected across hubs)
   * If the hide property has been populated in the manifest of the new app, the old XML add-in will get hidden in Outlook and end user will see only one app at a given point in time. 
   
3. Admin makes the new app available on Teams
   * Users would be able to acquire the new app / app update On Teams. Rest of the behavior would be same as scenario 1 above